test/fixtures/group-basic.json

```diff
@@ -18,10 +18,10 @@
   "output": {
     "user_id": "user-id",
     "time": 1388534400000,
-    "user_properties": {
+    "groups": {
```

Comment:
It is confusing because Github isn't picking it up as the order changing since both objects have the same value of: 

```
{
      "[Segment] Group": "0e8c78ea9d97a7b8185e8632"
    }
```

You can see the original [here](https://github.com/segment-integrations/integration-amplitude/blob/master/test/fixtures/group-basic.json#L21-L26)